tests/cli/conftest.py

```diff
@@ -38,6 +43,23 @@
 from nucypher.utilities.sandbox.constants import MOCK_CUSTOM_INSTALLATION_PATH_2, INSECURE_DEVELOPMENT_PASSWORD
 
 
+# CI machines don't have libusb available, thus usb1 raises an OSError.
+# This is a hack around that so we can patch what we need to run on CI.
+try:
+    import usb1
+except OSError:
+    class mock_usb1:
+
+        class USBErrorNoDevice(Exception):
+            pass
+
+        class USBErrorBusy(Exception):
+            pass
+
+    usb1 = mock_usb1()
+    sys.modules['usb1'] = usb1
```

Comment:
Hahah... Mocks like this always make me giggle.

Perhaps this can be moved to a test module outside of CLI?